app/src/main/kotlin/com/wire/android/notification/MessageNotificationManager.kt

```diff
@@ -486,7 +486,7 @@ class MessageNotificationManager
                 messagesStyle.addMessage(replyMessage)
             }
 
-            val notification = setUpNotificationBuilder(context, userId).apply {
+            val notification = setUpNotificationBuilder(context, userId, true).apply {
```

Comment:
`setOnlyAlertOnce(true)` should be enough, because it's the update of the notification, we pass the same id so it shouldn't alert it again when `setOnlyAlertOnce` is added, no need to create a separate channel 😄

Reply:
Didn't know about that 💪